src/Commands/TerminusCommand.php

```diff
@@ -21,21 +24,34 @@
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
+        $this->setLog(new ConsoleLogger($this->output()));
```

Comment:
Might want to inject a logger; I'd think you would definitely want to inject the Config() object, since setting that up probably involves parsing files.